rfc/rfc003-oauth2-authorization.md

```diff
@@ -34,9 +34,8 @@ In this document we will provide a way of protecting RESTful APIs with use of an
 * **Resource server**: The application \(a protected resource\) that requires authorized access to its API’s.
 * **JWT bearer token**: JWT encoded bearer token contains the user’s identity, subject and custodian and is signed by the acting party. This token is used to obtain an OAuth 2 access token.
 * **Access token**: An OAuth 2 access token, provided by an authorization Server. This token is handed to the client so it can authorize itself to a resource server. The contents of the token are opaque to the client. This means that the client does not need to know anything about the content or structure of the token itself.
-* **Authorization server**: The authorization server checks the user’s identity and credentials and creates the access token. The authorization server is trusted by the resource server. The resource server can exchange the access token for a JSON document with the user’s identity, subject, custodian, token validity and scope. This mechanism is called token introspection which is described by [RFC7662](https://tools.ietf.org/html/rfc7662).
-* **Request context**: The context of a request identified by the access token. The access token refers to this context. The context consists of the **custodian**, **actor**, **subject** and scope of the request like, in this case, an SSO. A context can be retrieved by performing token inspection.
-* **Endpoint reference**: every registered endpoint has a unique reference which is calculated as the hash of the registration document.
+* **Authorization server**: The authorization server checks the user’s identity and credentials and creates the access token. The authorization server is trusted by the resource server. The resource server can exchange the access token for a JSON document with the user’s identity, subject, custodian and token validity. This mechanism is called token introspection which is described by [RFC7662](https://tools.ietf.org/html/rfc7662).
+* **Request context**: The context of a request identified by the access token. The access token refers to this context. The context consists of the **custodian**, **actor**,  **Endpoint reference**: every registered endpoint has a unique reference which is calculated as the hash of the registration document.
```

Comment:
Comment not subject to this particular change but anyways; refer to RFC006 for endpoint definitions? (at least I think you're referring to the registry regarding endpoint registration document). In that case you can say it must contain the reference of the endpoint registration document as described by RFC006